tests/test_model_selection/test_backtest.py

```diff
@@ -11,48 +10,28 @@
 from etna.metrics import MSE
 from etna.metrics import SMAPE
 from etna.metrics.base import MetricAggregationMode
-from etna.model_selection.backtest import CrossValidationMode
-from etna.model_selection.backtest import TimeSeriesCrossValidation
-from etna.models.base import Model
 from etna.models.catboost import CatBoostModelMultiSegment
-from etna.models.linear import LinearPerSegmentModel
 from etna.models.prophet import ProphetModel
+from etna.pipeline.pipeline import CrossValidationMode
+from etna.pipeline.pipeline import Pipeline
 from etna.transforms import DateFlagsTransform
-from etna.transforms.base import Transform
 
 DEFAULT_METRICS = [MAE(mode=MetricAggregationMode.per_segment)]
 
 
-def test_repr():
-    """Check __repr__ method of TimeSeriesCrossValidation."""
-    model = LinearPerSegmentModel(fit_intercept=True, normalize=False)
-    mode = CrossValidationMode.expand.value
-    tscv = TimeSeriesCrossValidation(model=model, horizon=12, n_folds=3, metrics=DEFAULT_METRICS, mode=mode)
-    model_repr = model.__repr__()
-    metrics_repr_inner = ", ".join([metric.__repr__() for metric in DEFAULT_METRICS])
-    metrics_repr = f"[{metrics_repr_inner}]"
-    mode_repr = CrossValidationMode[mode].__repr__()
-    tscv_repr = tscv.__repr__()
-    true_repr = (
-        f"TimeSeriesCrossValidation(model = {model_repr}, horizon = 12, metrics = {metrics_repr}, "
-        f"n_folds = 3, mode = {mode_repr}, n_jobs = 1, )"
-    )
-    assert tscv_repr == true_repr
-
-
 @pytest.mark.parametrize("n_folds", (0, -1))
-def test_invalid_n_split(n_folds: int):
+def test_invalid_n_split(n_folds: int, example_reg_tsds):
```

Comment:
https://github.com/tinkoff-ai/etna/blob/master/tests/test_pipeline/test_pipeline.py#L181 
we already have all these tests in `tests/test_pipeline/test_pipeline.py`

Reply:
So I can just remove `test_model_selection/test_backtest.py`?